searx/engines/duckduckgo.py

```diff
@@ -74,6 +74,7 @@ def request(query, params):
 
     params['headers']['Content-Type'] = 'application/x-www-form-urlencoded'
     params['headers']['Referer'] = 'https://google.com/'
+    params['headers']['User-Agent'] = 'curl/7.81.0'
```

Comment:
```suggestion
    params['headers']['User-Agent'] = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:109.0) Gecko/20100101 Firefox/111.0'
```

Reply:
> Suggested change
>     params['headers']['User-Agent'] = 'curl/7.81.0'
>     params['headers']['User-Agent'] = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.14; rv:109.0) Gecko/20100101 Firefox/111.0'

It doesn't work for me.
I still receive the error, and I deactivated DDG.

Reply:
after changing the code you have to restart the uwsgi service

Reply:
> after changing the code you have to restart the uwsgi service

Of course! 😀